### DISCUSSION TITLE
Help with interface.

### DISCUSSION THREAD

**Discussion by Wundrwaffle** [ModelSurge/sd-webui-comfyui#85] [Q&A] [answered]
Hello. Did everything by hundreds of tutorials, but faced with problem. Seems footer loading earlier than ComfyUI workspace:
![Screenshot_3](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/536ba977-d61c-490e-8597-13b6be0d5725)

+ there are tab at main txt2img page:
![Screenshot_2](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/f40c6ec0-95d9-4c1d-9864-ea3c3bd1874b)

And after refresh page or reloading webui, comfyUI just disappearing and got this message: 
![Screenshot_4](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/6c980833-2229-4b75-bbf8-b2c466da722f)

Have to reload webui.bat to get back tabs and menu.

Help please to understand, what this problem came from? 
Google Chrome 115.0.5790.102

**Comment by ljleb** [comment]
I am confused why the source code is unavailable. Please provide us with the following informaion:
- which version of stable diffusion is this?
- which other extensions are active? does it work if comfyui is the only active extension
- wbich operating system are you using?

**Reply by ljleb** [reply]
Ah I know why reloading doesn't work. Working on a fix for that. I may be able to look into the odd window arrangement when we have more information about your setup.

**Comment by ljleb** [comment]
It looks like you are running on a relatively old version of stable diffusion. Please update the webui to a newer version, for example:
- 1.4.0
- 1.5.0

**Reply by ljleb** [reply]
I think we should intend to support webui>=1.5.1. We may drop support for older versions when it becomes impractical to support them.

**Comment by PladsElsker** [comment]
Also, once it is working properly, please refer to the wiki if you have questions about how to use ComfyUI workflows in the txt2img/img2img tabs. They are not exactly the same as the one in the ComfyUI Tab.

**Reply by PladsElsker** [reply]
Make sure to update ComfyUI as well if it's not up to date.

**Comment by Wundrwaffle** [comment]
Ok. Im using Automatic1111 commit: [22bcc7be]
Different extensions are active.
Win 10 is using.
![Screenshot_6](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/82533f97-1fa3-4781-a0d1-40881897084e)
![Screenshot_5](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/a1af05be-efbf-48a4-a6dd-fa6f23c424e0)

python: 3.10.6  •  torch: 1.13.1+cu117  •  xformers: 0.0.16rc425  •  gradio: 3.23.0  •  commit: [22bcc7be] •  checkpoint: [093d2bc8b6]

Tried to use ComfyUI at separate tab with different ip.
WebUI goes at http://127.0.0.1:7860 and ComfyUI goes at http://127.0.0.1:8189

Generated couple pics at comfy, then get back to webui and it crushed. 
After each approach to generate i got this:
![Screenshot_3](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/69259965-473d-4c1e-a98b-4cc657fb4b66)

Had to remove venv folder and switch off ComfyUI to bring webui back to life.

Extensions:
![Screenshot_7](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/c22998f5-de7f-4866-8dfe-e737bc16d5f3)

**Comment by PladsElsker** [comment, marked as answer]
current last commit hash of the webui is [a3ddf46](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/a3ddf464a2ed24c999f67ddfef7969f8291567be). 

I tried with your commit, I can't even start a generation, just stops, no errors. 

You should consider running `git pull` on all your git repos. A1111, This extension, controlnet as well as ComfyUI.

**Reply by Wundrwaffle** [reply]
Updated everything: 
![Screenshot_11](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/ac64f0a5-b44a-49aa-a147-18569eabb0f1)
Now everything working fine. Thank you.
![Screenshot_12](https://github.com/ModelSurge/sd-webui-comfyui/assets/117706507/8f8a9e63-e46d-41e1-a964-fa2940928010)
